packages/pdf/src/RPA/PDF/keywords/document.py

```diff
@@ -0,0 +1,510 @@
+import os
+import tempfile
+from pathlib import Path
+from typing import Any
+
+import pdfminer
+import PyPDF2
+from fpdf import FPDF, HTMLMixin
+from pdfminer.pdfdocument import PDFDocument
+from pdfminer.pdfparser import PDFParser
+from PIL import Image
+
+from RPA.core.helpers import required_param
+from RPA.core.notebook import notebook_print
+from RPA.PDF.keywords import (
+    LibraryContext,
+    keyword,
+)
+
+
+class PDF(FPDF, HTMLMixin):
+    pass
+
+
+class DocumentKeywords(LibraryContext):
+    """Keywords for basic PDF operations"""
+
+    def __init__(self, ctx):
+        super().__init__(ctx)
+        self.fpdf = PDF()
+        self._output_directory = Path(".")
+
+    @property
+    def output_directory(self):
+        return self._output_directory
+
+    @output_directory.setter
+    def output_directory(self, path: str):
+        self.output_directory = Path(path)
+
+    def __del__(self):
```

Comment:
Python will clean up file handles when the references go out of scope, and in general __del__ is a bit unpredictable. Would just remove it.